deploy/helm/prometheus-overrides.yaml

```diff
@@ -73,6 +75,8 @@ prometheusOperator:
   prometheus-node-exporter:
     ## Additional labels for pods in the DaemonSet
     podLabels: {}
+    ## Additional annotations for pods in the DaemonSet
+    podAnnotations: {}
```

Comment:
I can see `prometheus-node-exporter.podLabels` in the prometheus [docs](https://github.com/helm/charts/tree/master/stable/prometheus-operator) but cannot see `podAnnotations`. Are we sure is it implemented?

Reply:
Yikes! True that. Shall I just remove this or add a TODO to 
a) make a PR to https://github.com/helm/charts/tree/master/stable/prometheus-operator and then add it here
b) mark it that annotations for prometheus node exporter are not supported
?

Reply:
I would say:
1. Create an issue in our repo and in the prometheus-operator and link them.
2. Remove it from our code (because the issue will gather all the info)

Reply:
I've created an issue for that https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/825